src/components/implementation/torrent/ramfs/ramfs.c

```diff
@@ -23,6 +24,43 @@ struct fsobj root;
 #define UNLOCK() if (lock_release(&fs_lock)) BUG();
 
 #define MIN_DATA_SZ 256
+#define DEBUG 0
+
+struct file_data {
+	cbuf_t cbuf_id;
+	char *chunk_start;
+	int cbuf_len;
+	int start;
+	int len;
+	void* next;
+};
+
+int file_data_init(struct file_data **fd, 
```

Comment:
this style is inconsistent with the rest of the code-base.  I don't think that the style guide says anything about how to format function arguments.  I should add that.  To maintain consistency with what we have already, I think the rule should be that when argument lists need to be broken across lines, that the the subsequent lines should be lined up with the first line's argument list, and that each line should be roughly the same length.  I don't think that a separate line per argument is consistent with what's already in the code-base.  I don't have a strong argument for which makes the code more readable, but I do have a strong argument about what's consistent at this point.